dask/dataframe/io/io.py

```diff
@@ -570,10 +570,11 @@ def from_delayed(
 
     Parameters
     ----------
-    dfs : list of Delayed
+    dfs : list of Delayed or Future
         An iterable of ``dask.delayed.Delayed`` objects, such as come from
-        ``dask.delayed`` These comprise the individual partitions of the
-        resulting dataframe.
+        ``dask.delayed`` or an iterable of ``concurrent.futures.Future`` objects,
```

Comment:
We're talking about [Dask's implementation of `concurrent.futures`](https://docs.dask.org/en/stable/futures.html), so this should be:

"an iterable of `distributed.Future`"

Ref: https://distributed.dask.org/en/stable/api.html#future

Reply:
Thank you! for the insight @pavithraes, but just out of curiosity isn't it a good idea to use `dask.distributed.Future` here ?

Reply:
Just verified now that the distributed class is in another repo. So, we should indeed use `distributed.Future`. If possible, will you be able to provide insight on it for my better understanding?

Kindly excuse me for my incompetency.

Reply:
Understood @pavithraes, I will update the PR now to reflect the changes.